### ISSUE TITLE
Problems with the metagenome mode

### ISSUE THREAD

**Issue by ecollette** [bcgsc/NanoSim#102]
When simulating reads with the metagenome mode i've had some trouble with the fastq files generated.
It seems that the fourth line doesn't have the same number of bases than the second line which returns errors when trying to align with minimap:

e.g.
```
@reads:1
AATAATCGCGA
+
5,&7*5
```

here is the command i ran:
```
python ./NanoSim/src/simulator.py metagenome -gl ./sim_data/sample_config_file/metagenome_list_for_simulation\
      -a ./sim_data/sample_config_file/abundance_2.tsv\
      -dl ./sim_data/sample_config_file/dna_type_list.tsv\
      -c ./NanoSim/pre-trained_models/metagenome_ERR3152364_Even/training\
      -o ./sim_data/run2 -b albacore --fastq -t 5

```

**Comment by cheny19** [comment]
Hi @ecollette , we are aware of this problem and it is fixed in the latest v3.0.0-pre-release. However, I have just fixed another bug in that pre-release, so could you try to download the latest commit and give it a shot? Thanks!

**Comment by ecollette** [comment]
Just tried it and it works, thank you so much!
